src/aks-preview/azcli_aks_live_test/configs/ext_coverage_config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "totals": {
+        "fail_percent": 59,
```

Comment:
why is this different with cli_coverage_config.json

Reply:
For the current situation, the overall unit test coverage in acs (azure-cli) is merely above 70%. For aks-preview (azure-cli-extensions) the value is 59%+. There is a big gap between the two modules, so I am using two configuration files to control them separately.